components/swipe/swipeMouse.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Touch = ({ children, onEnd, onMove, onStart, ...rest }) => (
```

Comment:
The name — for touch events: Touch, I guess for mouse events should be Move (or so)?